test/datalog_tests/run-tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Runs all the tests in the current directory
+# If some arguments are given they are interpreted as tests to  run
+
+# Stop at first failure
+set -e
+
+# Extra compiler flags for some tests, indexed by test basename
+declare -A extraflags
+extraflags[three]="--output-internal-relations"
+
+# Detect directory where script resides
+mydir=$(dirname "$0")
+if [[ ! -d "$mydir" ]]; then mydir="$PWD"; fi
+# Always run in the directory where the script is installed
+cd ${mydir}
+
+function run_test {
+    local testname=$1;
+    local base=$(basename ${testname} .dl)
+
+    ddlog -i ${base}.dl -L../../lib ${extraflags[${base}]}
+    cd ${base}_ddlog
+    cargo build
+    cd ..
+    if [ -f ${base}.dat ]; then
+        ${base}_ddlog/target/debug/${base}_cli --no-print <${base}.dat >${base}.dump
+        if [ -f ${base}.dump.expected ]; then
+            diff -q ${base}.dump ${base}.dump.expected
+        elif [ -f ${base}.dump.expected.gz ]; then
+            gzip ${base}.dump
```

Comment:
I don't think this is necessary. `zdiff` can compare a compressed  file against an uncompressed one.